webform.tokens.inc

```diff
@@ -297,3 +283,28 @@ function webform_tokens($type, $tokens, array $data = array(), array $options =
 
   return $replacements;
 }
+
+/**
+ * Provides default webform token modifier behavior.
+ */
+function webform_token_modifier_defaults($modifier, $display_element) {
+  switch ($modifier) {
+    case 'label':
+      return webform_filter_xss($display_element['#title']);
+
+    case 'withlabel':
+      return render($display_element);
+
+    case 'nolabel':
+      $display_element['#theme_wrappers'] = array();
+      return render($display_element);
+
+    case 'key':
+      $values = array();
+      foreach ($display_element['#value'] as $value) {
+        $values[] = webform_filter_xss($value);
+      }
+      return implode(' ', $values);
+  }
+  return t('BAD MODIFIER');
```

Comment:
Maybe better to just return an empty string.

Also, we should add `@param string $modifier`, `@param array $display_element` and `@return string` to the docblock.

Reply:
Both done, thanks.